civihr_default_theme/scss/components/_ui-select.scss

```diff
@@ -76,6 +76,17 @@
 * TEMP fixes till ui-select styling is fixed
 * Dropdown ul opacity was set to 0
 */
-.chr_leave-request-modal__form .ui-select-container ul {
-  opacity: 1 !important;
+.chr_leave-request-modal__form {
+ .ui-select-container ul {
+   opacity: 1 !important;
+  }
+
+  .dropdown-menu {
+    border: 1px solid #ccc;
```

Comment:
First of all I think adding new color is not necessary and we can use the same color of the input box.(Pending confirmation from @AkA84 ) If we decide to add new color, it should be defined in the variables. No colors should be used without variables, that is what we are following in civihr. Also you can search for a already defined color which is close to `#ccc` and use the same.